core/src/main/java/io/stargate/core/activator/BaseActivator.java

```diff
@@ -121,6 +161,11 @@ private synchronized void startServiceInternal() {
     }
     started = true;
 
+    if (healthCheck != null) {
+      String healthCheckName = activatorName.toLowerCase();
```

Comment:
is this `toLowerCase` required? Is there a problem to have health-checks not lowercased?

Reply:
not really required, but I thought it would be nice for consistency since they are used as keys in URLs. I can remove the conversion to lower case if this is a concern.